locales/bn.json

```diff
@@ -327,6 +331,10 @@
     "message": "বিঃদ্রঃ: এটি কেবল পঠনযোগ্য বাকেট। আপনার ব্রাউজার উইন্ডোটি বন্ধ করার পরে আপনি যে পরিবর্তনগুলি করেন তা অক্ষত থাকবে না।",
     "description": "Paragraph of body text"
   },
+  "javascript.console.readonly#alert@2": {
+    "message": "দ্রষ্টব্য: এটি একটি রিড-ওনলি বালতি; আপনার সম্পাদনা অস্থায়ী। আপনার সম্পাদনা স্থায়ী করতে, বালতি নাম ক্লিক করে বালতি ফর্ক করুন:",
```

Comment:
Change this to this :arrow_right:  `"message": "দ্রষ্টব্য: এটি একটি রিড-ওনলি বাকেট; আপনার সম্পাদনা অস্থায়ী। আপনার সম্পাদনা স্থায়ী করতে, বাকেট নাম ক্লিক করে বাকেট ফর্ক করুন:"`